packages/migrations/src/types.ts

```diff
@@ -27,3 +28,38 @@ export enum ContractName {
     EtherDelta = 'EtherDelta',
     Arbitrage = 'Arbitrage',
 }
+
+export interface LedgerCommunicationClient {
+    close: () => Promise<void>;
+}
```

Comment:
This type is exported by the `@0xproject/subproviders` package, don't copy it over here.

```
import { LedgerCommunicationClient } from '@0xproject/subproviders';
```